templates/zerver/help/format-your-message-using-markdown.md

```diff
@@ -62,8 +62,8 @@ Numbered lists
 
 ## Links
 
-Zulip auto-linkifies URLs and valid stream names. You can also add a
-[custom linkifier](/help/add-a-custom-linkifier) to link
+Zulip auto-linkifies URLs and valid stream (and topic) names. You can also
```

Comment:
I think maybe it'd be work having this link to the page about linking to a conversation.